Enable Banking.postman_collection.json

```diff
@@ -0,0 +1,282 @@
+{
+	"info": {
+		"_postman_id": "d8fb687d-7b9f-4155-a58e-a6b1c7197f8a",
+		"name": "Enable Banking",
+		"description": "This is collection of sample requests to Tilisy.com.\nEach request needs JWT token, which generates in Pre-request Script. For using this, add your RSA private key to collection Variables (variable name - 'private_key').\nIf you have a key in .pem format, run:\n```openssl rsa -in private.pem -out private.key```",
```

Comment:
to api.tilisy.com. 

Instead of "If you have a key in .pem format, run:\n```openssl rsa -in private.pem -out private.key```". I'd put "Go to https://enablebanking.com/cp/applications and register your application".

Reply:
fixed